packages/template-retail-rsc-app/src/app/components/productGrid/index.tsx

```diff
@@ -3,10 +3,26 @@ import type {ShopperSearchTypes} from 'commerce-sdk-isomorphic'
 import ProductCard from '@/app/components/productCard'
 
 export default function ProductGrid({
-    products
+    products,
+    isLoading = false
```

Comment:
@gurpreetsainisalesforce Where is the `isLoading` coming from? With server components we should not actually need this. Which hints at AI hallucinated code 😶.

Reply:
> @gurpreetsainisalesforce Where is the `isLoading` coming from? With server components we should not actually need this. Which hints at AI hallucinated code 😶.

I'll have to create a new PR on the new repo. Lets see if that will be any better :)